utils.js

```diff
@@ -0,0 +1,76 @@
+function generateSummary(core, metrics) {
+  body = '### Lava scan results\n\n';
+
+  if (Object.keys(metrics.vulnerability_count).length === 0) {
+    return body + `No vulnerabilities found!! 🎉🎉🎉\n\n`;
```

Comment:
I would use one single exclamation mark, emoji and newline.

```suggestion
    return body + `No vulnerabilities found! 🎉\n`;
```

Reply:
done